foundation_cms/campaigns/models/campaign_page.py

```diff
@@ -79,7 +82,7 @@ class CampaignPage(AbstractBasePage):
     )
 
     thank_you_image = models.ForeignKey(
-        "wagtailimages.Image",
+        get_image_model_string(),
```

Comment:
Updated with this so we can use the image chooser field on the CMS! 
<img width="3840" height="1756" alt="image" src="https://github.com/user-attachments/assets/503049fc-ee8b-478f-8b86-c810efad43e2" />